src/formatters.jl

```diff
@@ -25,11 +25,11 @@ Ex) "[{level} | {name}]: {msg}" will print message of the form
 struct DefaultFormatter <: Formatter
     fmt_str::AbstractString
     tokens::Vector{Pair{Symbol, Bool}}
-    output_tz::Dates.TimeZone
+    output_tz::Union{Dates.TimeZone, Nothing}
```

Comment:
I wonder if we should have a timezone in TimeZones.jl called `User` or `UserLocal`,
which has the behavour of that having this as `nothing` does; where it is a cached version of `localzone`.

Reply:
Yeah, I was talking to Curtis a bit about it, but I'd have to think more about how it should work and if that has some potential downfalls. Looks like this caching behaviour might be causing issues on Linux.